packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -19,33 +19,38 @@ export interface ImportOptions {
 }
 
 export abstract class ImportBase {
-  public abstract get moduleName(): string;
-  protected abstract async generateTypeScript(code: CodeMaker): Promise<void>;
+  public abstract get moduleNames(): string[];
+  protected abstract async generateTypeScript(code: CodeMaker, moduleName?: string): Promise<void>;
 
   public async import(options: ImportOptions) {
     const code = new CodeMaker();
-    const fileName = `${this.moduleName}.ts`;
-    code.openFile(fileName);
-    code.indentation = 2;
-    await this.generateTypeScript(code);
-    code.closeFile(fileName);
 
     const outdir = path.resolve(options.outdir);
     await fs.mkdirp(outdir);
+    const isTypescript = options.targetLanguage === Language.TYPESCRIPT
 
-    if (options.targetLanguage === Language.TYPESCRIPT) {
-      await code.save(outdir);
-      return;
-    } 
+    for (const name of this.moduleNames) {
+      const fileName = `${name}.ts`;
+      code.openFile(fileName);
+      code.indentation = 2;
+      await this.generateTypeScript(code, name);
+      code.closeFile(fileName);
+    
+      if (isTypescript) {
+        await code.save(outdir);
+      } 
+    }
+
+    if (isTypescript) return
 
     // this is not typescript, so we generate in a staging directory and harvest the code
     await withTempDir('importer', async () => {
       await code.save('.');
-      await jsiiCompile('.', {
-        main: this.moduleName,
-        name: this.moduleName,
-      });
 
+      await this.moduleNames.forEach(name => jsiiCompile('.', {
+        main: name,
+        name,
+      }))
```

Comment:
I rather avoid `forEach` in async code:

```suggestion
for (const moduleName of this.moduleNames) {
  await jsiiCompile('.', { main: moduleName, name: moduleName });
}
```

Reply:
Yeah learned something new about async `forEach`s while doing this. Missed one, will fix it!